client/src/main/java/org/evosuite/testcase/execution/ExecutionResult.java

```diff
@@ -29,13 +29,18 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.Serializable;
 import java.util.*;
 import java.util.stream.IntStream;
 
-public class ExecutionResult implements Cloneable {
+public class ExecutionResult implements Cloneable, Serializable {
```

Comment:
The class "RottenGreenTests" (i.e., one of the implemented test smells) has the following line of code:
`ExecutionResult lastExecutionResult = chromosome.getLastExecutionResult();`

where "chromosome" is of type "TestChromosome". Originally, in "ExecutableChromosome.java" (where the "getLastExecutionResult()" method is implemented), we had the following field:
`protected transient ExecutionResult lastExecutionResult = null;`

We were having problems because the variable "lastExecutionResult" in "RottenGreenTests.java" was null. Therefore, it was necessary to make 2 changes:

1 - In "ExecutableChromosome.java", remove "transient" from the field:
`protected ExecutionResult lastExecutionResult = null;`

2 - Make ExecutionResult Serializable